includes/Modules/Analytics.php

```diff
@@ -41,22 +41,22 @@
 use Google\Site_Kit\Modules\Analytics\Web_Tag;
 use Google\Site_Kit\Modules\Analytics\Proxy_AccountTicket;
 use Google\Site_Kit\Modules\Analytics\Advanced_Tracking;
-use Google\Site_Kit_Dependencies\Google_Service_Analytics;
-use Google\Site_Kit_Dependencies\Google_Service_AnalyticsReporting;
-use Google\Site_Kit_Dependencies\Google_Service_AnalyticsReporting_GetReportsRequest;
-use Google\Site_Kit_Dependencies\Google_Service_AnalyticsReporting_ReportRequest;
-use Google\Site_Kit_Dependencies\Google_Service_AnalyticsReporting_Dimension;
-use Google\Site_Kit_Dependencies\Google_Service_AnalyticsReporting_DimensionFilter;
-use Google\Site_Kit_Dependencies\Google_Service_AnalyticsReporting_DimensionFilterClause;
-use Google\Site_Kit_Dependencies\Google_Service_AnalyticsReporting_DateRange;
-use Google\Site_Kit_Dependencies\Google_Service_AnalyticsReporting_Metric;
-use Google\Site_Kit_Dependencies\Google_Service_AnalyticsReporting_OrderBy;
-use Google\Site_Kit_Dependencies\Google_Service_Analytics_Accounts;
-use Google\Site_Kit_Dependencies\Google_Service_Analytics_Account;
-use Google\Site_Kit_Dependencies\Google_Service_Analytics_Webproperties;
-use Google\Site_Kit_Dependencies\Google_Service_Analytics_Webproperty;
-use Google\Site_Kit_Dependencies\Google_Service_Analytics_Profile;
-use Google\Site_Kit_Dependencies\Google_Service_Exception;
+use Google\Site_Kit_Dependencies\Google\Service\Analytics as Google_Service_Analytics;
```

Comment:
Thanks, @ivankruchkoff. This PR looks great to me from a technical point of view. However, after updating classes from Google SDK to follow PSR-4 format, it is hard to distinguish SDK classes and classes created in the plugin. It may make a negative impact in the future when a new engineer ramps up on the project and needs to understand our codebase.

Can I ask you to update your PR to use aliases everywhere for SDK classes as you do it here?

So, instead of this:

```diff
- use Google\Site_Kit_Dependencies\Google_Service_Analytics_Accounts;
+ use Google\Site_Kit_Dependencies\Google\Service\Analytics\Accounts;

  ...

  if ( 'User does not have any Google Analytics account.' === $wp_error->get_error_message() ) {
-     return new Google_Service_Analytics_Accounts();
+     return new Accounts();
  }
```

Let's do this everywhere:

```diff
- use Google\Site_Kit_Dependencies\Google_Service_Analytics_Accounts;
+ use Google\Site_Kit_Dependencies\Google\Service\Analytics\Accounts as Google_Service_Analytics_Accounts;

  ...

  if ( 'User does not have any Google Analytics account.' === $wp_error->get_error_message() ) {
      return new Google_Service_Analytics_Accounts();
  }
```